### PR TITLE
Update Cloud Databases panel login links to use https and clarify wording.

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by RackerKate** [comment]
@cyrichardson Please review - ok to go now.

**Comment by cyrichardson** [comment]
@RackerKate On this one now.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5b13752eb1/how-to/).

File: content/cloud-databases/managing-backups-for-cloud-databases.md [hunk 1]
```diff
@@ -27,9 +27,8 @@ automatic backups of your databases. For more information, see
 To create a backup of a database instance, use the following steps:
 
 1. Log in to the [Cloud Control Panel](https://login.rackspace.com/).
-2. Click the **Select a Product** drop-down menu in the top navigation bar,
-   then select **Rackspace Cloud**.
-2. In the top navigation bar, select **Databases > MySQL**.
+2. In the top navigation bar, click **Select a Product > Rackspace Cloud**.
+2. Select **Databases > MySQL**.
```
Comment (catlook) [inline content/cloud-databases/managing-backups-for-cloud-databases.md]: line numbering?  two #2s

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6ed2773ff0/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ba2d0723d4/how-to/). :bow:

**Pull request merged**